### PR TITLE
feat: display contributors

### PR DESCRIPTION
**Pull request by AurelienLourot**
@brillout this looks like this. What do you think?

Screenshot from 2024-01-16 16-11-17

### CODE REVIEW

**Comment by brillout** [comment]
Love it. Every pixel of it.

Shall we CC everyone?

LGTM: feel free to merge & release.

**Comment by brillout** [comment]
Idea (feel free to ignore): also adding a section mentioning that some team members offer consulting.

```diff
      <HorizontalLine />
      <Contributors />
+     <HorizontalLine />
+     <Consulting />
```

But maybe the `counsutling` button is enough 🤔, not sure.

(Feel free to ignore if you feel like you have a better plan.)

**Comment by AurelienLourot** [comment]
@magne4000 @nitedani @phonzammi we'd like to highlight maintainers and contributors on https://vike.dev/ as depicted in the screenshot above. Are you fine with the way it is? You can modify/remove yourself at any time by creating a PR to `maintainersList.tsx`, or also feel free to modify the react components if you'd like to show different things about yourself.

For context, Vike's documentation is built on this project "docpress", which in built on Vike.

**Comment by brillout** [comment]
Maybe a TLDR (I guess one/two sentences?) of `vike.dev/counsutling`. I'm inclined to list the counsulting people in `vike.dev/counsutling`, otherwise I'm thinking this may be a little bit redundant with the previous section.

**Comment by phonzammi** [comment]
Thanks so much, I don't know what to say. 
It's up to you guys, whenever you think that I don't deserve this, you can just remove/move my name.  
Anyway this is awesome, I love it 😍

**Comment by magne4000** [comment]
That's really nice! :+1: with the way it is

**Comment by brillout** [comment]
Some suggestions, all optional (as always).

 - Maybe a call-to-action button. Similar to the `Sponsor` button.

image


 - I think `For issues` sounds simpler than `For concrete issues`.
 - Maybe `the Vike team offers consulting` instead of `some of us...` to increase conversion rate.

As always, feel free to ignore my suggestions.

**Comment by brillout** [comment]
> * Maybe `the Vike team offers consulting` instead of `some of us...` to increase conversion rate.

Maybe a different text between the TLDR and the consulting page 🤔

**Comment by AurelienLourot** [comment]
Updated:

Screenshot from 2024-01-16 20-36-28

> Maybe a different text between the TLDR and the consulting page 🤔

Yes good idea. I'll have a better text in Vike's `consulting.page.server.mdx`. For here in the DocPress Demo I think it's fine to use the same text.

**Comment by brillout** [comment]
👍

(Text brainstorming: `For advanced help, the Vike team also offers consulting`.)

**Comment by brillout** [comment]
I rewrote the consulting TDLR a bit, but I'm not sure if it's better. Feel free to revert / change.

File: src/components/Consulting.tsx [hunk 1]
```diff
@@ -7,20 +7,21 @@ import { SupporterSection, SectionDescription, CallToAction } from './Supporters
 import { Maintainer } from './Contributors'
 import { maintainers } from './maintainersList'
 import { Link } from './Link'
+const consultingPageHref = '/consutling'
 
-function Consulting({ consultingPageHref }: { consultingPageHref: string }) {
```
Comment (AurelienLourot) [inline src/components/Consulting.tsx]: @brillout do you mind if I revert? I wanted the consuming project (Vike or something else) to have the freedom to specify any `/page` and not be forced to have a `/consulting` page.

Also there is a typo two lines above (`/consutling`)

**Review by AurelienLourot — COMMENTED** [review]
LGTM now, what do you think?

File: src/components/Consulting.tsx (hunk 1 above)
Comment (brillout) [reply src/components/Consulting.tsx]: Yes that's what I understood, but using docpress other than for Vike and Telefunc isn't a goal. Maybe we'll use it for other projects at some point, but I don't think we should ever make it a public project like Vitepress. That's why I think it's pretty safe to assume that the page will have the URL `/consulting`.

There are a lot of assumptions in other parts of the code like that. I think we'll make our lives easier if we keep these assumptions.

> Also there is a typo two lines above (`/consutling`)

Sorry about that, I'll double check in the future.

**Comment by nitedani** [comment]
You can also mention vike-angular under my name

**Comment by AurelienLourot** [comment]
> You can also mention vike-angular under my name

@nitedani I can't make up my mind whether it's a good idea or not to have more than three points for each maintainer. Definitely the list isn't meant to be complete (we could for sure come up with more points for @brillout and @magne4000 too, for example). So there is a trade-off to be found between a list which is very incomplete and cards that would be way too big. What do you think? Would you like me to insert `vike-angular` *instead* of another point? Or would you prefer having four points?

File: src/components/Consulting.tsx (hunk 1 above)
Comment (AurelienLourot) [reply src/components/Consulting.tsx]: @brillout ok I'll re-revert-ish, stay tuned

Comment (AurelienLourot) [reply src/components/Consulting.tsx]: done

**Comment by AurelienLourot** [comment]
> Would you like me to insert `vike-angular` _instead_ of another point?

I have simply replaced `vike-react-zustand` (which doesn't exist yet) with `vike-angular` for now in @nitedani's list. Let me know what you think.

**Comment by nitedani** [comment]
Looks good 👍

**Comment by brillout** [comment]
@4350pChris Would you be up for being part of that list?

**Comment by 4350pChris** [comment]
> @4350pChris Would you be up for being part of that list?

I'd be happy to be on that.

**Comment by AurelienLourot** [comment]
:+1: done. It looks like this:

Screenshot from 2024-01-18 15-31-03

**Pull request merged**